lib/VixDiskLib/VixDiskLib.rb

```diff
@@ -91,7 +91,7 @@ def self.exit
   def self.setup_env
     vars_to_clear = %w(BUNDLE_BIN BUNDLE_BIN_PATH BUNDLE_GEMFILE
                        BUNDLE_ORIG_MANPATH EVMSERVER GEM_HOME
```

Comment:
@jerryk55 Your comment states: "...GEM_HOME to GEM_PATH as two variables that will no longer be cleared" but it looks as if GEM_HOME is still in the list of the env being cleared. Please clarify.